pkg/platform/provider/baremetal/cluster/kubeadm.go

```diff
@@ -177,8 +178,8 @@ func (p *Provider) getAPIServerExtraArgs(c *v1.Cluster) map[string]string {
 		"token-auth-file": constants.TokenFile,
 	}
 	if p.config.Audit.Address != "" {
-		args["audit-policy-file"] = constants.AuditPolicyConfigFile
-		args["audit-webhook-config-file"] = constants.AuditWebhookConfigFile
+		args["audit-policy-file"] = path.Join(constants.KubernetesDir, constants.AuditPolicyConfigFile)
```

Comment:
Why use path.Join?

Reply:
because these const also used by other [code](https://github.com/tkestack/tke/blob/33bf207cd2303881de934d38ff20a29f2fef09a4/pkg/platform/provider/baremetal/cluster/create.go#L476), so I remove the prefix path and use path.join

Reply:
Add another const will be better, avoid use join.Later may be change.

Reply:
fixed